topology.go

```diff
@@ -13,7 +15,8 @@ var (
 	// The table that keeps names of the generations changed names.
 	// This is a list of known supported names, starting from the newest one.
 	generationsTableNames []string = []string{
-		"system_distributed.cdc_streams", // Introduced in Scylla 4.1
+		"system_distributed.cdc_streams_descriptions", // Introduced in Scylla 4.3
+		"system_distributed.cdc_streams",              // Introduced in Scylla 4.1
 		"system_distributed.cdc_description",
```

Comment:
no need to support older versions I think
(many other things have changed by the way)

Reply:
Good point, removed.